src/main/java/duke/commands/CompleteCommand.java

```diff
@@ -22,8 +22,8 @@ public void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException
         } else {
             tasks.get(index).markAsDone();
             System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
```

Comment:
Bot have been shifted to GUI version, maybe dont require print statements anymore?